locales/en-US/experiments.ftl

```diff
@@ -58,13 +57,13 @@ containersContributors7Title = Firefox UX
 containersContributors8Title = Firefox UX
 containersContributors9Title = Firefox UX
 minvidDescription = Keep videos front and center. Min Vid lets you display YouTube and Vimeo videos in a small frame that stays in the foreground as you browse the web.
-minvidIntroduction = <p>Love multitasking? Try Min Vid. It’s like picture-in-picture where one picture is a media player and the other is the entire Web.</p><ul><li><strong>Click and go:</strong> Pop up the Min Vid frame to keep videos playing in the foreground while you browse.</li><li><strong>Put it where you want it:</strong> Min Vid lets you drag the video frame anywhere on your desktop.</li><li><strong>Controls where you need them:</strong> Min Vid includes controls to let you pause/play, scrub, drag, minimize, and adjust volume right in the frame.</li><li><strong>Now supporting audio playback:</strong> Min Vid now supports playback from SoundCloud and direct mp3 links.</li><li><strong>Create a queue:</strong> Min Vid now supports queues, just start up a video and add tracks from any supported domains (YouTube, Vimeo, SoundCloud, and direct links).</li><li><strong>View your history:</strong> Min Vid keeps track of your history, just pop open the queue panel from the top right of the player and you can access all the media you've been playing.</li></ul>
+minvidIntroduction = <p>Love multitasking? Try Min Vid. It’s like picture-in-picture where one picture is a media player and the other is the entire Web.</p> <ul> <li><strong>Click and go:</strong> Pop up the Min Vid frame to keep videos playing in the foreground while you browse.</li> <li><strong>Put it where you want it:</strong> Min Vid lets you drag the video frame anywhere on your desktop.</li> <li><strong>Controls where you need them:</strong> Min Vid includes controls to let you pause/play, scrub, drag, minimize, and adjust volume right in the frame.</li> <li><strong>Now supporting audio playback:</strong> Min Vid now supports playback from SoundCloud and direct mp3 links.</li> <li><strong>Create a queue:</strong> Min Vid now supports queues, just start up a video and add tracks from any supported domains (YouTube, Vimeo, SoundCloud, and direct links).</li> <li><strong>View your history:</strong> Min Vid keeps track of your history, just pop open the queue panel from the top right of the player and you can access all the media you've been playing.</li></ul>
```

Comment:
Looks like the only thing that changed is spacing between markups, so this one should be OK.